### PR TITLE
Docs Typo

### PR DESCRIPTION
**Pull request by michaelKaefer**

### CODE REVIEW

**Comment by montogeek** [comment]
I don't think it is a typo. cc @EugeneHlushko

**Comment by wizardofhogwarts** [comment]
It isn't a typo, but this improvement is required.

**Comment by EugeneHlushko** [comment]
It says: Inline as a data URI; inlining *to* a data URI does sound weird to me. If it said “transform to X” i would like the change. Why do you think this is required? Please explain

File: src/content/concepts/loaders.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ contributors:
   - furkle
 ---
 
-Loaders are transformations that are applied on the source code of a module. They allow you to pre-process files as you `import` or “load” them. Thus, loaders are kind of like “tasks” in other build tools and provide a powerful way to handle front-end build steps. Loaders can transform files from a different language (like TypeScript) to JavaScript or inline images as data URLs. Loaders even allow you to do things like `import` CSS files directly from your JavaScript modules!
+Loaders are transformations that are applied on the source code of a module. They allow you to pre-process files as you `import` or “load” them. Thus, loaders are kind of like “tasks” in other build tools and provide a powerful way to handle front-end build steps. Loaders can transform files from a different language (like TypeScript) to JavaScript or inline images to data URLs. Loaders even allow you to do things like `import` CSS files directly from your JavaScript modules!
```
Comment (anikethsaha) [inline src/content/concepts/loaders.mdx]: This is one glitch I noticed. But still both `on` and `to` looks fine to me 👍 

```suggestion
Loaders are transformations that are applied to the source code of a module. They allow you to pre-process files as you `import` or “load” them. Thus, loaders are kind of like “tasks” in other build tools and provide a powerful way to handle front-end build steps. Loaders can transform files from a different language (like TypeScript) to JavaScript or inline images to data URLs. Loaders even allow you to do things like `import` CSS files directly from your JavaScript modules!
```

[the remark this answers is not in the document]
Comment (michaelKaefer) [reply src/content/concepts/loaders.mdx]: You are totally right. I think this is much better.

[the remark this answers is not in the document]
Comment (michaelKaefer) [reply src/content/concepts/loaders.mdx]: Done

**Comment by michaelKaefer** [comment]
I changed my commit and committed both suggestions from @anikethsaha. I think that now everybody can agree that it is an improvement.

**Comment by anikethsaha** [comment]
cc @EugeneHlushko

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**